### ISSUE TITLE
Drawer doesn't restrict focus when open

### ISSUE THREAD

**Issue by tjsavage** [PolymerElements/paper-drawer-panel#145]
Safari on Mac OS X and FireFox on Windows X

Pre-requisites: 
Enable VoiceOver on Safari (COMMAND F5) or lanunch NVDA on Windows
1. Go to 
   https://elements.polymer-project.org/bower_components/paper-drawer-panel/demo/index.html
2. With the TAB key, move to the "toggle drawer" button and press ENTER. 
3. Press TAB to move through items in the drawer. 

Observed behavior: 
Please note, when you reach last item in the drawer and press the TAB key, focus moves all the way up to the browsers' tabs. It moves through all open tabs and other things located on the browser toolbar, and only then moves back to the drawer. This seems to be a regression. 

Expected behavior: 
pressing tab inside the drawer should circle between the items in the drawer or ding at the end and not move anywhere. In either case, focus should be restricted to the drawer.

**Comment by blasten** [comment]
We already have that behavior in app-drawer (app-layout) https://elements.polymer-project.org/bower_components/app-layout/demo/demo6.html. (toggle device mode). I'd recommend to use app-layout.
